src/utils/auth.js

```diff
@@ -0,0 +1,38 @@
+import { BASE_URL } from "./Api";
+
+// SIGNUP
+export const response = (res) => {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`checkResponse`